src/GraphQLMiddleware.php

```diff
@@ -9,12 +9,6 @@
 
 class GraphQLMiddleware
 {
-
-    /**
-     * @var string The graphql uri path to match against
-     */
-    private $graphql_uri = "/graphql";
```

Comment:
this can be set, has an hard deps, in constructor by the factory. taken from config...